sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -55,6 +55,62 @@ template <> uint32_t pi_cast(uint64_t Value) {
   std::terminate();
 }
 
+// Returns the ze_structure_type_t to use in .stype of a structured descriptor.
+// Intentionally not defined; will give an error if no proper specialization
+template <class T> ze_structure_type_t getZeStructureType();
```

Comment:
why only these? the number of stypes are more than these ones.

https://github.com/oneapi-src/level-zero/blob/551dd5810a3cea7a7e26ac4441da31878e804b53/include/ze_api.h#L230

Reply:
We don't yet use other structures in the plugin. Will extend specializations as needed.